src/renderer/src/lib/song.ts

```diff
@@ -59,7 +59,6 @@ export function createDefaultSong(): Song {
     path: "",
     audio: "",
     bg: "",
-    dir: "",
```

Comment:
Why is dir being removed?

Reply:
dir is not in the "Song" type 
and i did not find it being used anywhere
![image](https://github.com/user-attachments/assets/afd82abc-37ac-4d95-9821-d7ac9952ff23)
![image](https://github.com/user-attachments/assets/e5b58b4c-e897-4c4c-8e36-6faa11ed0d99)